apps/vs-code-designer/src/app/commands/createNewProject/createProjectSteps/ZipFileStep.ts

```diff
@@ -0,0 +1,69 @@
+import { unzipLogicAppArtifacts } from '../../../utils/taskUtils';
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as os from 'os';
+import * as fs from 'fs';
+import { localize } from '../../../../localize';
+import { AzureWizardPromptStep } from '@microsoft/vscode-azext-utils';
+import type { IAzureQuickPickItem } from '@microsoft/vscode-azext-utils';
+import type { IFunctionWizardContext } from '@microsoft/vscode-extension-logic-apps';
+import * as rimraf from 'rimraf';
+
+export class ZipFileStep extends AzureWizardPromptStep<IFunctionWizardContext> {
+  private zipContent: Buffer | Buffer[];
+  private targetDirectory: string;
+  private static zipFilePath: string;
+  private wizardContext: IFunctionWizardContext;
+
+  public hideStepCount = true;
+  public supportsDuplicateSteps = false;
+  ui: any;
```

Comment:
NIT: Is this needed?

Reply:
hideStepCount is needed because the steps go from 3/6 to 4/4 because I am skipping the WorkFlowStep, but the rest are not needed.

Reply:
ohhh sorry, I meant the `ui: any;`